scripts/generate_psa_constants.py

```diff
@@ -395,6 +401,8 @@ def generate_psa_constants(header_file_names, output_file_name):
 if __name__ == '__main__':
     if not os.path.isdir('programs') and os.path.isdir('../programs'):
         os.chdir('..')
+    # Allow to change the directory where psa_constant_names_generated.c is written to.
+    OUTPUT_FILE_DIR = sys.argv[1] if len(sys.argv) == 2 else "programs/psa"
```

Comment:
[PEP8](https://www.python.org/dev/peps/pep-0008/#id45) says all-caps for global constants, lowercase for local variables. Please don't use all-caps for a local variable. (I'm surprised that Pylint didn't complain, it's usually picky about these things.)

Reply:
I originally put it lower case but the [CI compained](https://travis-ci.org/github/ARMmbed/mbedtls/jobs/692210799#L1646):
```
scripts/generate_psa_constants.py:400:4: C0103: Constant name "output_file_dir" doesn't conform to UPPER_CASE naming style (invalid-name)
```
This variable is not declared under a function so maybe it is still a global variable?

Reply:
Oh, I see, it isn't in a function, there's no `main` function in this script. Ok, let's keep it like this for now.